tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -91,3 +126,120 @@ def list_databases(self):
         database_names = [db['name'] for db in databases]
 
         return database_names
+    
+    @staticmethod
+    def _prep_data_or_file(file: str = None, data: list = None) -> pd.DataFrame:
+        """Prepares the data from memory or loaded from disk for addition or deletion action.
+
+        :param file: path to the file to be prepared
+        :type file: str
+        :param data: data to be prepared
+        :type data: list
+
+        :returns: prepared data in dataframe format
+        :rtype: pd.DataFrame
+        """
+        if not bool(file) and not bool(data):
+            print('No data to import. Specify a file location or a dataset.')
+        
+        if bool(file):
+            return utils.read_file(file=file)
+          
+        if bool(data):
+            return utils.to_csv(data=data)
+    
+    def add(self, schema: str, table: str, file: str = None, data: list = None):
```

Comment:
Ah, I now see your endpoint, I think you are actually executing 'save' (i.e. add or update).

Reply:
propose you rename 'add' to 'save'

Reply:
`add` has been renamed to `save`